mzLib/Omics/ISpectrumMatch.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Omics.Fragmentation;
+
+namespace Omics
+{
+    public interface ISpectrumMatch
```

Comment:
I think this could be a class which PsmFromTsv inherits from. That way we can move the methods in PsmFromTsv to the base SpectrumMatch class